drift/lib/src/sentry_span_helper.dart

```diff
@@ -65,91 +49,71 @@ class SentrySpanHelper {
     }
   }
 
-  /// This span is used for the database transaction.
-  @internal
-  ISentrySpan? transactionSpan;
-
-  /// @nodoc
-  @internal
   T beginTransaction<T>(
-    String description,
     T Function() execute, {
     String? dbName,
   }) {
-    final currentSpan = _hub.getSpan();
-    final span = currentSpan?.startChild(
-      SentryQueryExecutor.dbOp,
-      description: description,
+    final scopeSpan = _hub.getSpan();
+    _parentSpan = scopeSpan?.startChild(
```

Comment:
Does it make a difference if we assign `_parentSpan` sooner? Before we did it in the finally block. Also, why the rename to `_parentSpan` from `transactionSpan`, was quite fitting, no?

Also one question, are nested transactions possible and is this a case we should handle with some kind of span stack?

Reply:
> Does it make a difference if we assign _parentSpan sooner?

no, I only added it to the top for visibility

> why the rename to _parentSpan from transactionSpan

the new implementation of spanhelper manages the parentSpan to which to attach the instrumented operations to, previously we would just keep track of the transactionSpan (drift transaction span, not sentry transaction) which was correct but now the span isn't always a drift transaction span. if you dont start a drift transaction the span a simple crud would attach the span to the scope span which is not a drift transaction span

Reply:
> nested transactions possible and is this a case we should handle with some kind of span stack?

nested transactions are supported by drift, but I didn't think of this

we should support for it imo, I'll have a look